server/protocols/jmap-rfc-8621/.gitignore

```diff
@@ -0,0 +1,15 @@
+logs
```

Comment:
We dont commit local setup in the project gitignore (only global build result). So no idea and related. Please use a local git ignore for that purpose.

Reply:
Oh, im sorry. Fixing it.